### PR TITLE
Clean up Heroku application manifest

### PR DESCRIPTION
**Pull request by gjvis**
Clean up `/app.json`:
- remove unused and redundant environment variables
- remove unused addons
- specify environment variable defaults and generators appropriately
- use more obvious buildpack url
- add github repo

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
   },
   "env": {
-    "EMAIL_SENDER"             : { "required" : true  },
-    "GOOGLE_TAG_MANAGER_ID"    : { "required" : true  },
-    "LANG"                     : { "required" : true  },
-    "RACK_ENV"                 : { "required" : true  },
-    "RAILS_ENV"                : { "required" : true  },
-    "RAILS_SERVE_STATIC_FILES" : { "required" : true  },
-    "SECRET_KEY_BASE"          : { "required" : true  }
+    "EMAIL_SENDER":             { "required": true },
```
Comment (nathancashmoredwp) [inline app.json]: Is this required based on the bits removed on the other commit ?  Or have i looked at the requests in the wrong order and you've already cleaned this out.?

File: app.json [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
+    "GOOGLE_TAG_MANAGER_ID":    { "required": true },
+    "RAILS_ENV":                { "required": true, "value": "production" },
```
Comment (nathancashmoredwp) [inline app.json]: Does this mean the default value is production ?

File: app.json (hunk 1 above)
Comment (gjvis) [inline app.json]: whoops!

File: app.json (hunk 2 above)
Comment (gjvis) [inline app.json]: Yes

**Comment by nathancashmoredwp** [comment]
:bikini:

**Pull request merged**